view/photo.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="/public/css/photo.css">
+</head>
+<body>
+    <nav>
+        <ul class="nav-list">
+            <li> <a href="/view/home.html"> HOME </a></li>
+            <li> <a href="/view/works.html">Works </a></li>
+            <li><a href="/view//about.html">About</a></li>
+
+        </ul>
+    </nav>
+    <h1>Peggy Gou</h1>
+    <div class="container">
+        <div class="col-grid">
```

Comment:
In the `app.js` file you already specify that every static elements that you will be using in your website will be located inside the `public` folder:

```
app.use(express.static("public"));
```

So when you need to reference the `path` of something that is inside the `public` folder you don't need to specify public in the path you just need to right the path after the public folder:

```
        <div class="col-grid">
            <img src="/images/image1.webp" >
            <img src="/images/image2.jpeg" >
            <img src="/images/image3.jpeg">
            <img src="/images/image4.png">
            <img src="/images/image5.jpeg">
        </div>
```